src/Compiler/Backend/X64/CodeGenX64.sml

```diff
@@ -1167,6 +1166,7 @@ struct
                           | Blockf64_sub_real => blockf64_sub_real (b,x,y,d,size_ff,C)
                           | Blockf64_update_f64 => blockf64_update_f64 (b,x,y,d,size_ff,C)
                           | M256d_plus => die "M256d_plus: boxed operation not implemented"
+                            | F256_box => f256_box_kill_tmp01 (b,x,y,d,size_ff,C)
```

Comment:
Consider renaming F256_box to F256_assign and make it instead assign into already allocated storage...